signac/core/synced_collection.py

```diff
@@ -133,6 +157,17 @@ def load(self):
             else:
                 self._parent.load()
 
+    @classmethod
+    def _validate(cls, data):
+        """Validate the input data."""
+        # Validate for every parent class which have _validators
+        for _cls in cls.mro():
```

Comment:
I _think_ we want to use `cls.__mro__` instead, because it's the cached result of `cls.mro()` at the time the class is instantiated (not an _instance_ of the class, in my understanding). (The `mro` method is defined by `type`.) [https://docs.python.org/3/library/stdtypes.html#class.\_\_mro\_\_](https://docs.python.org/3/library/stdtypes.html#class.__mro__)
```suggestion
        for _cls in cls.__mro__:
```